src/main/java/io/anserini/search/SimpleImpactSearcher.java

```diff
@@ -119,6 +123,27 @@ public SimpleImpactSearcher(String indexDir) throws IOException {
     cascade.add(new ScoreTiesAdjusterReranker());
   }
 
+  /**
+   * Sets the query encoder
+   * 
+   * @param encoder the query encoder
+   */
+  public void set_onnxQueryEncoder(String encoder) {
```

Comment:
Should we use standard Python naming conventions here? e.g., `set_onnx_query_encoder`?